roles/swift-ring/handlers/main.yml

```diff
@@ -4,22 +4,3 @@
            --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
   become: True
   become_user: swiftops
-
-- name: setup swift rings - scaleup
```

Comment:
I am not ready to move this in yet. I would like to take a hard look at swifttool in Q3.